lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -63,9 +80,79 @@ def _get_config(self, config_dict):
                                config_dict['ovs_internal_sampling_port_number'],
             internal_sampling_fwd_tbl=
                               config_dict['ovs_internal_sampling_fwd_tbl_number'],
+            multi_tunnel_flag=multi_tunnel_flag,
 
         )
 
+    def _ovs_multi_tunnel_init(self):
+        try:
+            subprocess.check_output("sudo ovs-vsctl list Open_vSwitch | grep gtpu",
+                                     shell=True,)
+            self.ovs_gtp_type = "gtpu"
+        except subprocess.CalledProcessError:
+            self.ovs_gtp_type = "gtp"
+
+        try:
+            subprocess.check_output('ovs-vsctl set-manager %s' % OVSDB_MANAGER_ADDR,
+                                     shell=True,)
+        except subprocess.CalledProcessError as e:
+            logging.warning("Error for set manager with ovsdb: %s", e)
+
+    def _ip_addr_to_gtp_port_name(self, enodeb_ip_addr:str):
+        ip_no = hex(int(ipaddress.ip_address(enodeb_ip_addr)))
+        buf = "g_{}".format(ip_no[2:])
+        return buf
+
+    def _get_ovs_bridge(self, dpid):
+        ovsdb_addr = 'tcp:%s:%d' % (self._datapath.address[0], OVSDB_PORT)
```

Comment:
The common function was moved to utility module and currently the flag "ovs_multi_tunnel" set as a False because Gnb address is not yet support for uplink (PDR).